phc/phc_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIfaceInfoToPHCDevice(t *testing.T) {
+	info := &EthtoolTSinfo{
+		PHCIndex: 0,
+	}
+	got, err := ifaceInfoToPHCDevice(info)
+	require.NoError(t, err)
+	require.Equal(t, "/dev/ptp0", got)
+
+	info.PHCIndex = 23
+	got, err = ifaceInfoToPHCDevice(info)
+	require.NoError(t, err)
+	require.Equal(t, "/dev/ptp23", got)
+
+	info.PHCIndex = -1
```

Comment:
If you execute this against `lo` is it `-1`?

Reply:
yep